schema/crdb/vpc-subnet-contention/up02.sql

```diff
@@ -0,0 +1,80 @@
+set local disallow_full_table_scans = off;
```

Comment:
Just a drive-by comment here for other reviewers - this schema migration looks good to me, even though it has multiple statements, because they're all "data modifying" statements (SELECT/UPDATE/DELETE) rather than "data definition" statements which would alter tables/indices.

Looks like the rule of "max of one data-definition statement" per file is being respected here.